Updates — Configuration Service (#256)
commerce-docs/microsite-commerce-storefront: src/content/docs/setup/configuration/commerce-configuration.mdx

<Vocabulary>
### Storefront configuration

> A JSON object which maps properties and values used by the Commerce boilerplate.

Reviewer: ```suggestion
A JSON object that maps properties and values used by the Commerce boilerplate.
```

> The storefront configuration is a JSON object which lives in either [config-service](https://www.aem.live/docs/config-service-setup) and contains the connection settings for your Commerce blocks. The boilerplate repo also contains a `demo-config.json` file which can be renamed to `config.json` in lieu of using the config service.

Reviewer: ```suggestion
The storefront configuration is a JSON object which lives in either [config-service](https://www.aem.live/docs/config-service-setup) and contains the connection settings for your Commerce blocks. The boilerplate repo also contains a `demo-config.json` file that can be renamed to `config.json` instead of using the config service.
```

### default values

By default, the values in the `demo-config.json` file are from the boilerplate's sample backend to ensure everything works out of the box. But when it comes time to connect your own backend, you need to know what each key means so you can update it with the correct value from your own Commerce instance.

### getConfigValue function

The `getConfigValue` function is a helper function that retrieves the `value` from the `configs` file using the dot-notation path as an argument. For example, `getConfigValue('headers.cs.x-api-key')` returns the value of `[root config].headers.cs.x-api-key`.

### getHeaders function

The `getHeaders` function is a helper function which takes a storefront scope (like `cs`, `cart`, etc) and reads values from the configs file with the corresponding `header` keys and returns an object map of those rows. For example, if you have properties in config like `headers.cs.Magento-Environment-Id": "abc-123"`, then `getHeaders('cs')` will return an object like `{ "Magento-Environment-Id": "abc-123"}`.

</Vocabulary>

## Examples

> You can find the `config` file in your code repo at either `/config.json` if you created it, or `demo-config.json`. Open the file to view the connection **keys** and **values** of the sample backend. They should look similar (but not exact) to the following:

Reviewer: ```suggestion
You can find the config file in your code repo at either /config.json or /demo-config.json. You can view the sample backend's connection `keys` and `values` in the file. They should look similar to this:
```
